src/registrar/views/domain_request.py

```diff
@@ -182,7 +183,9 @@ def configure_step_options(self):
 
     def has_pk(self):
         """Does this wizard know about a DomainRequest database record?"""
-        return "domain_request_id" in self.storage
+        if self.kwargs.get("id") is not None:
```

Comment:
does this work the same if we one liner this to `return self.kwargs.get("id")` or `return self.kwargs.get("id") is not None`?